uint8 id_element
int32 value
bool debug